kubernetes/windows/setup.ps1

```diff
@@ -74,7 +74,7 @@ Write-Host ('Finished Extracting Certificate Generator Package')
 $windowsazuremonitoragent = [System.Environment]::GetEnvironmentVariable("WINDOWS_AMA_URL_NEW", "process")
 if ([string]::IsNullOrEmpty($windowsazuremonitoragent)) {
     Write-Host ('Environment variable WINDOWS_AMA_URL is not set. Using default value')
-    $windowsazuremonitoragent = "https://github.com/microsoft/Docker-Provider/releases/download/windows-ama-bits/genevamonitoringagent.46.17.2.zip"
+    $windowsazuremonitoragent = "https://github.com/microsoft/Docker-Provider/releases/download/windows-ama-bits/genevamonitoringagent.46.31.3.zip"
```

Comment:
@zanejohnson-azure  - Can you please add following details to the PR ...

1. scenarios are validated
2. resource usage of the windows agent before and after
3.  Review the errors and warnings in windows ama agent logs